src/components/KRadio/KRadio.vue

```diff
@@ -198,6 +212,8 @@ const kRadioClasses = computed((): Record<string, boolean> => {
     'input-error': props.error,
     checked: isChecked.value,
     'has-description': showDescription.value,
+    'card-horizontal': props.card && props.cardOrientation === 'horizontal',
+    'card-vertical': props.card && props.cardOrientation === 'vertical',
```

Comment:
issue: Since `vertical` is the default, you need to edit this logic a bit to ensure it is applied if the user passes an invalid value.
```suggestion
    'card-horizontal': props.card && props.cardOrientation === 'horizontal',
    // Add vertical class for `vertical` or an invalid prop value
    'card-vertical': props.card && props.cardOrientation !== 'horizontal',
```